serialize_property_test.ts

```diff
@@ -380,3 +380,72 @@ test({
     );
   },
 });
+
+test({
+  name: "Serialize key function",
+  fn() {
+    class Test1 extends Serializable {
+      @SerializeProperty(
+        (
+          (propertyName) => "_" + (propertyName as string)
```

Comment:
This cast is kind of dirty, I think a better way of handling it would be to throw on `Symbol`.

Reply:
```suggestion
          (propertyName) => `_${String(propertyName)}`
```

I believe this calls `.toStrnig` on Symbol